pwndbg/integration/binja.py

```diff
@@ -153,13 +153,24 @@ def init_bn_rpc_client() -> None:
 
 
 def with_bn(fallback: K = None) -> Callable[[Callable[P, T]], Callable[P, T | K]]:
+    global _bn
+
     def decorator(func: Callable[P, T]) -> Callable[P, T | K]:
+        global _bn
+
         @functools.wraps(func)
         def wrapper(*args: P.args, **kwargs: P.kwargs) -> T | K:
+            global _bn
```

Comment:
Does this really require global x3? I think only this one is needed?

Reply:
i tried but it didn't work without it, just python stuff i guess